configs/frontend_config/clinical-visits.json

```diff
@@ -0,0 +1,88 @@
+{
+  "@openmrs/esm-patient-chart-app": {
+    "extensionSlots": {
+      "clinical-visits-slot": {
+        "add": [
+          "encounter-list-table-tabs"
+        ],
+        "configure": {
+          "encounter-list-table-tabs": {
+            "tabDefinitions": [
+              {
+                "tabName": "clinicalVisit",
+                "headerTitle": "clinicalVisit",
+                "displayText": "clinicalVisitEncounters",
+                "encounterType": "cb0a65a7-0587-477e-89b9-cf2fd144f1d4",
+                "columns": [
+                  {
+                    "id": "clinicalVisitDate",
+                    "isDate": true,
+                    "title": "visitDate",
+                    "concept": "163137AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "isLink": true
+                  },
+                  {
+                    "id": "clinicalVisitType",
+                    "title": "visitType",
+                    "concept": "8a9809e9-8a0b-4e0e-b1f6-80b0cbbe361b"
+                  },
+                  {
+                    "id": "clinicalScreeningOutcome",
+                    "title": "tBScreeningOutcome",
+                    "concept": "160108AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                  },
+                  {
+                    "id": "clinicalNextAppointmentDate",
+                    "title": "nextAppointmentDate",
+                    "isDate": true,
+                    "concept": "5096AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+                  },
+                  {
+                    "id": "actions",
+                    "title": "actionsTitle",
+                    "actionOptions": [
+                      {
+                        "formName": "POC Clinical Visit Form v2",
+                        "package": "hiv-care-treatment",
+                        "label": "viewDetails",
+                        "mode": "view"
+                      },
+                      {
+                        "formName": "POC Clinical Visit Form v2",
+                        "package": "hiv-care-treatment",
+                        "label": "editForm",
+                        "mode": "edit"
+                      }
+                    ]
+                  }
+                ],
+                "launchOptions": {
+                  "displayText": "add",
+                  "moduleName": ""
```

Comment:
What is this for?

Reply:
Something where we used to specific module names but that shouldn't be important now, I have removed it.